src/containers/CryptoList/index.tsx

```diff
@@ -0,0 +1,43 @@
+import List from '../../components/List';
+import {useAppDispatch, useAppSelector} from '../../hooks/reduxHooks';
+import {fetchCryptoWithSymbol} from '../../store/actions';
+import {
+  AddCryptoText,
+  AvatarImg,
+  Title,
+  TopBar,
+  TouchableAddCrypto,
+} from './styles';
+import React from 'react';
+import {FlatList, View} from 'react-native';
+
+const CryptoList = () => {
+  const {cryptos: fetchCrypto} = useAppSelector(state => state);
```

Comment:
Maybe is better to do the destructuring inside the useAppSelector

useAppSelector(({ cryptos }) => cryptos)